stable/node-problem-detector/README.md

```diff
@@ -91,7 +91,7 @@ helm install my-release oci://ghcr.io/deliveryhero/helm-charts/node-problem-dete
 | settings.custom_plugin_monitors | list | `[]` |  |
 | settings.extraArgs | list | `[]` |  |
 | settings.heartBeatPeriod | string | `"5m0s"` | Syncing interval with API server |
-| settings.log_monitors | list | `["/config/kernel-monitor.json","/config/docker-monitor.json"]` | User-specified custom monitor definitions |
+| settings.log_monitors | list | `["/config/kernel-monitor.json", "/config/docker-monitor.json", "/config/readonly-monitor.json"]` | User-specified custom monitor definitions |
```

Comment:
I don't see how this is failing the docs CI check 🤔

Reply:
I think there's an extra space